.github/workflows/gradle.yml

```diff
@@ -0,0 +1,31 @@
+name: Java CI
```

Comment:
build.gradle should be updated:
https://github.com/hmcts/idam-java-client/blob/master/build.gradle#L14-L17

travis.yml removed: https://github.com/hmcts/send-letter-client/blob/master/.travis.yml

Reply:
update as suggested.